build.gradle

```diff
@@ -38,12 +38,20 @@ repositories {
 dependencies {
     compile 'org.slf4j:slf4j-api:1.7.12'
 
-    def tapestry_version='5.3.7'
+    def tapestry_version='5.5.0-alpha-1'
 
     compile "org.apache.tapestry:tapestry-jpa:${tapestry_version}"
     compile 'javax.enterprise:cdi-api:1.2'
 
-    testCompile 'junit:junit:4.12'
+		testCompile 'org.testng:testng:6.8.21'
+
+    testRuntime "org.apache.tapestry:tapestry-test:${tapestry_version}"
+		// JpaModule doesn't run without servlet-api, perhaps something we should address later
+		testRuntime 'javax.servlet:javax.servlet-api:3.0.1'
+
+		testRuntime 'org.eclipse.persistence:eclipselink:2.5.1'
+		testRuntime 'com.h2database:h2:1.3.175'
+
```

Comment:
Yes. Tapestry uses spaces. You seemed to have mixed formatting, so I didn't worry about formatting at the moment. All code needs to be reformatted accoding to Tapestry formatting conventions when merged.